components/button/button.js

```diff
@@ -42,6 +42,7 @@ class Button extends ButtonMixin(LitElement) {
 				button {
 					font-family: inherit;
 					padding: 0.55rem 1.5rem;
+					position: relative;
```

Comment:
I have to admit, requiring consumers to do this doesn't feel great. If we change the tooltip positioning logic in the future, these `position: relative` styles may become unnecessary, and it will be a little risky to remove them if we aren't certain they aren't needed for some other purpose as well.

I'm not really clear on what it is in the tooltip positioning logic that requires the target to be positioned, but I wonder if there is a fix we can make to that logic that doesn't require consumers to do this.